plotly/offline.py

```diff
@@ -0,0 +1,149 @@
+""" Plotly Offline
+    A module to use Plotly's graphing library with Python
+    without connecting to a public or private plotly enterprise
+    server.
+"""
+from __future__ import absolute_import
+
+from plotly import utils
+import uuid
+import json
+import os
+import requests
+
+
+def download_plotlyjs(download_url, plotlyjs_dir='~/.plotly/plotlyjs'):
```

Comment:
Finally, if this didn't require any arguments, we could just call it the first time a user tries to use it, which would mean they don't need to know about it.